SU-SOE/stanford_soe_helper
# 7.x 2.x collections soe 2715 cjw

# READY FOR REVIEW

# Summary
- This implements the Collections list view

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this.


# Steps to Test
- Verify that stanford_soe_helper_collection_views and stanford_soe_helper_collection_layouts modules are not overridden
- Switch to this branch
- May need to revert these feature modules
- Navigate to `magazine/collections`
- Verify you see the collection page

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2736

## Related PRs
Branched from:
https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-collections-SOE-2715
## More Information




# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
